src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1449,8 +1451,6 @@ void fromNetwork (
     using namespace std::chrono_literals;
     auto constexpr RPC_NOTIFY = 10min;
 
-    auto j = logs.journal ("HTTPClient");
-
     HTTPClient::request (
```

Comment:
We can change `HTTPClient::request` (and the two variants of `request`) to not take a `Logs&` as their last argument; all they do is the pass it to the constructor of `HTTPClientImp` which uses it to call `logs.journal ("HTTPClient")`. So why not pass that to begin with?